src/sif/motorscootercost.cc

```diff
@@ -285,12 +286,14 @@ class MotorScooterCost : public DynamicCost {
    * @param  node  Node (intersection) where transition occurs.
    * @param  pred  the opposing current edge in the reverse tree.
    * @param  edge  the opposing predecessor in the reverse tree
+   * @param  has_measured_speed Do we have any of the measured speed types set?
    * @return  Returns the cost and time (seconds)
    */
   virtual Cost TransitionCostReverse(const uint32_t idx,
                                      const baldr::NodeInfo* node,
                                      const baldr::DirectedEdge* pred,
-                                     const baldr::DirectedEdge* edge) const override;
+                                     const baldr::DirectedEdge* edge,
+                                     const bool has_measured_speed) const override;
```

Comment:
Is `has_measured_speed` unsed here? If so, maybe make it an unnamed param to avoid compiler warning.

Reply:
Same comment for all other places where we have this param unused.

Reply:
ignore these too. sorry about the noise!

Reply:
@mandeepsandhu updated